analyzers/src/SonarAnalyzer.Common/Helpers/SonarAnalysisContext.cs

```diff
@@ -160,6 +166,32 @@ private static IEnumerable<XElement> ParseXmlSettings(SourceText sourceText)
             }
         }
 
+        private static bool IsTestProject(TryGetValueDelegate<ProjectConfigReader> tryGetValue, Compilation c, AnalyzerOptions options) =>
```

Comment:
_Code orientation:_ Let's group this `IsTestProject` with the others above.

Reply:
Ah, I missed that it's private. But let's keep as it is. We'll fix that in the future.